module/VuFindSearch/tests/unit-tests/src/VuFindTest/Backend/Solr/Document/UpdateDocumentTest.php

```diff
@@ -52,7 +52,8 @@ class UpdateDocumentTest extends TestCase
      */
     public function testAsXML()
     {
-        $record = $this->getMockForAbstractClass(\VuFindSearch\Backend\Solr\Record\SerializableRecordInterface::class);
+        $record = $this->getMockBuilder(\VuFindSearch\Backend\Solr\Record\SerializableRecordInterface::class)
+                        ->getMock();
```

Comment:
I think this can be simplified:

```suggestion
        $record = $this->createMock(\VuFindSearch\Backend\Solr\Record\SerializableRecordInterface::class);
```